src/mlpack/methods/ann/init_rules/variance_scaling_init.hpp

```diff
@@ -0,0 +1,202 @@
+/**
+ * @file variance_scaling_init.hpp
+ * @author Shashank Shekhar
+ *
+ * Intialization rule for the neural networks.
+ * VarianceScalingNormalInit is performed by assigning a gaussian matrix with
+ * zero mean and variance given by (scaling factor / n) to the weight matrix.
+ * VarianceScalingUniformInit is performed by assigning a random matrix between
```

Comment:
Do you think an extra line here could improve the overall readability?

Reply:
yes. I will add it.